GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1928,6 +1934,28 @@ void AddRecentRepositories(Repository repo, string caption)
             }
         }
 
+        private void OpenGitExRepository(string repoPath)
```

Comment:
```suggestion
        private void OpenRepo(string repoPath)
```
The repo is not specific to GE.